chalice/deploy/deployer.py

```diff
@@ -555,7 +579,13 @@ def _build_lambda_function(self,
         # type: (...) -> models.LambdaFunction
         function_name = '%s-%s-%s' % (
             config.app_name, config.chalice_stage, name)
-        return models.LambdaFunction(
+        security_group_ids = config.security_group_ids
```

Comment:
Mentioned this before but I think we can remove lines 582-586 because they are handled in the `_get_vpc_params()` method.

Reply:
This is fixed now.